src/main/java/org/opentripplanner/netex/loader/util/AbstractHierarchicalMap.java

```diff
@@ -38,10 +38,10 @@ public boolean containsKey(K key) {
     }
 
     /** Get value from 'local' map, parent is not queried. */
-    protected abstract V localGet(K key);
+    abstract V localGet(K key);
 
     /** Check if key exist in 'local' map, parent is not queried. */
-    protected abstract boolean localContainsKey(K key);
+    abstract boolean localContainsKey(K key);
 
```

Comment:
Removed `protected` because this abstract class is only implemented by classes in the same package - no need to allow other classes access.